browser/ui/content_settings/brave_widevine_content_setting_bubble_model.cc

```diff
@@ -8,17 +8,43 @@
 #include "brave/browser/ui/brave_browser_content_setting_bubble_model_delegate.h"
 #include "brave/common/pref_names.h"
 #include "brave/components/brave_shields/common/brave_shield_constants.h"
+#include "brave/grit/brave_generated_resources.h"
 #include "chrome/browser/component_updater/widevine_cdm_component_installer.h"
 #include "chrome/browser/content_settings/host_content_settings_map_factory.h"
 #include "chrome/browser/plugins/plugin_utils.h"
 #include "chrome/browser/profiles/profile_manager.h"
 #include "chrome/browser/subresource_filter/chrome_subresource_filter_client.h"
-#include "brave/grit/brave_generated_resources.h"
 #include "components/content_settings/core/browser/host_content_settings_map.h"
 #include "components/prefs/pref_service.h"
+#include "third_party/widevine/cdm/buildflags.h"
 #include "ui/base/l10n/l10n_util.h"
 
-BraveWidevineContentSettingPluginBubbleModel::BraveWidevineContentSettingPluginBubbleModel(
+#if BUILDFLAG(BUNDLE_WIDEVINE_CDM)
+#include <string>
+
+#include "base/bind.h"
+#include "brave/browser/widevine/brave_widevine_bundle_manager.h"
+#include "chrome/browser/lifetime/application_lifetime.h"
+#endif
+
+namespace {
+#if BUILDFLAG(BUNDLE_WIDEVINE_CDM)
+void OnWidevineInstallDone(const std::string& error) {
+  if (!error.empty()) {
+    LOG(ERROR) << __func__ << ": " << error;
+    return;
+  }
+
+  DVLOG(1) << __func__ << ": Widevine install completed w/o error";
+  // TODO(simonhong): This is very aggresive.
+  // Showing relaunch dialog would be better.
+  chrome::AttemptRelaunch();
```

Comment:
@bbondy I think there are many paths to make user restart.
1. just restart after install is success like I did
2. change content settings bubble string after install
3. show relaunch dialog

Which one is best UX? Or please suggest another one.

Reply:
@simonhong #2 please.  That messaging only for Linux.

Reply:
@bbondy UX is changed to #2. PTAL again.